.github/workflows/lint.yml

```diff
@@ -52,11 +57,21 @@ env:
   # the subset of development dependencies installed.
   INSTALL_EXTRA: lint
 
+  ORG_REPO: trailofbits/.github
+  ORG_REPO_REF: ${{ inputs.org-repo-ref }}
+  ORG_REPO_PATH: /tmp/org_github
+
 jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{env.ORG_REPO}}
+          ref: ${{env.ORG_REPO_REF}}
+          path: tmp/org_repo
+      - run: mv tmp/org_repo ${ORG_REPO_PATH}
```

Comment:
Out of curiosity: why is this necessary? Can't we just clone directly to `ORG_REPO_PATH`?

Reply:
TIL `actions/checkout` doesn't allow the `path:` to be outside of the runner's directory!